sdk/maps/maps-search/review/maps-search.api.md

```diff
@@ -0,0 +1,684 @@
+    Tesla = "Tesla"
+}
+
+// @public
+export enum KnownEntryPointType {
+    // (undocumented)
+    Main = "main",
+    // (undocumented)
+    Minor = "minor"
+}
+
+// @public
+export enum KnownGeographicEntityType {
+    // (undocumented)
+    Country = "Country",
+    // (undocumented)
+    CountrySecondarySubdivision = "CountrySecondarySubdivision",
+    // (undocumented)
+    CountrySubdivision = "CountrySubdivision",
+    // (undocumented)
+    CountryTertiarySubdivision = "CountryTertiarySubdivision",
+    // (undocumented)
+    Municipality = "Municipality",
+    // (undocumented)
+    MunicipalitySubdivision = "MunicipalitySubdivision",
+    // (undocumented)
+    Neighbourhood = "Neighbourhood",
+    // (undocumented)
+    PostalCodeArea = "PostalCodeArea"
+}
+
+// @public
+export enum KnownLocalizedMapView {
+    AE = "AE",
+    AR = "AR",
+    Auto = "Auto",
+    BH = "BH",
+    IN = "IN",
+    IQ = "IQ",
+    JO = "JO",
+    KW = "KW",
+    LB = "LB",
+    MA = "MA",
+    OM = "OM",
+    PK = "PK",
+    PS = "PS",
+    QA = "QA",
+    SA = "SA",
+    SY = "SY",
+    Unified = "Unified",
+    YE = "YE"
+}
+
+// @public
+export enum KnownMatchType {
+    // (undocumented)
+    AddressPoint = "AddressPoint",
+    // (undocumented)
+    HouseNumberRange = "HouseNumberRange",
+    // (undocumented)
+    Street = "Street"
+}
+
+// @public
+export enum KnownOperatingHoursRange {
+    NextSevenDays = "nextSevenDays"
+}
+
+// @public
+export enum KnownQueryType {
+    Global = "NON_NEAR",
+    Nearby = "NEARBY"
+}
+
+// @public
+export enum KnownRoadUseType {
+    // (undocumented)
+    Arterial = "Arterial",
+    // (undocumented)
+    LimitedAccess = "LimitedAccess",
+    // (undocumented)
+    LocalStreet = "LocalStreet",
+    // (undocumented)
+    Ramp = "Ramp",
+    // (undocumented)
+    Rotary = "Rotary",
+    // (undocumented)
+    Terminal = "Terminal"
+}
+
+// @public
+export enum KnownSearchAddressResultType {
+    // (undocumented)
+    AddressRange = "Address Range",
+    // (undocumented)
+    CrossStreet = "Cross Street",
+    // (undocumented)
+    Geography = "Geography",
+    // (undocumented)
+    POI = "POI",
+    // (undocumented)
+    PointAddress = "Point Address",
+    // (undocumented)
+    Street = "Street"
+}
+
+// @public
+export enum KnownSearchIndexes {
+    Address = "Addr",
+    CrossStreets = "Xstr",
+    Geographies = "Geo",
+    PointAddresses = "PAD",
+    PointsOfInterest = "POI",
+    Streets = "Str"
+}
+
+export { LatLon }
+
+// @public
+export type LocalizedMapView = string;
+
+// @public
+export class MapsSearchClient {
+    constructor(credential: AzureKeyCredential, options?: MapsSearchClientOptions);
+    constructor(credential: TokenCredential, mapsAccountClientId: string, options?: MapsSearchClientOptions);
+    beginFuzzySearchBatch(requests: FuzzySearchRequest[], options?: FuzzySearchBatchOptions): Promise<BatchPoller<BatchResult<SearchAddressResult>>>;
```

Comment:
I wonder if we should call these `queries` instead of `requests`. @jeremymeng what do you think?

Reply:
That seems to be consistent with the non-batch functions

Reply:
But the `requests` is in the array-type of `FuzzySearchRequest`, which contains not only `SearchQuery` but also another property `options`. Would it be better to keep it `requests` to avoid confusion?